java/rsi-example/src/main/java/rsi/example/amqp/Listener.java

```diff
@@ -0,0 +1,102 @@
+package rsi.example.amqp;
+
+import oracle.rsi.PushPublisher;
+import oracle.rsi.ReactiveStreamsIngestion;
+import oracle.sql.json.OracleJsonFactory;
+import oracle.sql.json.OracleJsonObject;
+import org.apache.qpid.jms.JmsConnectionFactory;
+import rsi.example.common.Retailer;
+import rsi.example.common.RSIService;
+
+import javax.jms.*;
+import java.io.ByteArrayInputStream;
+
+public class Listener {
+  private static final String ACTIVEMQ_USER = "admin";
+  private static final String ACTIVEMQ_PASSWORD = "password";
+  private static final String ACTIVEMQ_HOST = "localhost";
+  private static final int ACTIVEMQ_PORT = 5672;
+  private static final String TOPIC_NAME = "event";
+
+  private static final String DB_URL = "jdbc:oracle:thin:@" +
+      "(description= (retry_count=20)(retry_delay=3)(address=(protocol=tcps)(port=1521)(host=adb.us-phoenix-1.oraclecloud.com))(connect_data=(service_name=gebqqvpozhjbqbs_azuretestdb_high.adb.oraclecloud.com))(security=(ssl_server_dn_match=yes)))";
```

Comment:
This one should be templated to allow users to fill in their own connection details.
Certainly, we should not have a pre-existing service name in our examples.